Unable to build project

Hi 
I'm newbie in type script . I'm following the readme.md but it does not generate final js.

I'm calling 

```
$ npm install --dev
$ npm install typescript -g
$ npm run build  <!---- fails here
```

```
Condition always true [./~/jquery/dist/jquery.js:9778,0]

ERROR in /Users/i303813/Code/paas/__POC__/zipkin-ui/typings/globals/dagre-d3/index.d.ts
(5,44): error TS2305: Module 'Dagre' has no exported member 'DagreFactory'.

ERROR in /Users/i303813/Code/paas/__POC__/zipkin-ui/typings/globals/dagre-d3/index.d.ts
(17,32): error TS2503: Cannot find namespace 'd3'.

ERROR in /Users/i303813/Code/paas/__POC__/zipkin-ui/typings/globals/dagre-d3/index.d.ts
(17,52): error TS2503: Cannot find namespace 'd3'.

ERROR in /Users/i303813/Code/paas/__POC__/zipkin-ui/typings/globals/dagre-d3/index.d.ts
(23,57): error TS2503: Cannot find namespace 'd3'.

ERROR in /Users/i303813/Code/paas/__POC__/zipkin-ui/typings/globals/dagre-d3/index.d.ts
(23,100): error TS2305: Module 'Dagre' has no exported member 'Edge'.

ERROR in /Users/i303813/Code/paas/__POC__/zipkin-ui/typings/globals/dagre-d3/index.d.ts
(25,21): error TS2503: Cannot find namespace 'd3'.
Child html-webpack-plugin for "index.html":
```
My setup 

```
__POC__/zipkin-ui (master)$npm version
{ 'zipkin-ui': '0.1.0',
  npm: '3.10.7',
  ares: '1.10.1-DEV',
  http_parser: '2.7.0',
  icu: '57.1',
  modules: '48',
  node: '6.6.0',
  openssl: '1.0.2h',
  uv: '1.9.1',
  v8: '5.1.281.83',
  zlib: '1.2.8' }
```

--- a/typings.json
+++ b/typings.json
@@ -2,11 +2,10 @@
   "globalDependencies": {
     "bootstrap": "registry:dt/bootstrap#3.3.5+20160619023404",
     "core-js": "registry:dt/core-js#0.0.0+20160602141332",
+    "d3": "registry:dt/d3#0.0.0+20160907005744",
+    "dagre": "registry:dt/dagre#0.7.0+20160505164908",
     "dagre-d3": "registry:dt/dagre-d3#0.0.0+20160620231311",
     "jasmine": "registry:dt/jasmine#2.2.0+20160621224255",
     "node": "registry:dt/node#6.0.0+20160807145350"
-  },
-  "dependencies": {
-    "d3": "registry:npm/d3#3.0.0+20160723033700"
   }
 }

